doc/api/swagger.json

```diff
@@ -4363,6 +4363,216 @@
           }
         }
       }
+    },
+    "/tags": {
+      "get": {
+        "description": "Returns tags visible to current user according to given parameters",
+        "tags": [
+          "tags"
+        ],
+        "security": [
+          {
+            "basicAuth": []
+          }
+        ],
+        "produces": [
+          "application/json"
+        ],
+        "responses": {
+          "200": {
+            "description": "Tags returned",
+            "schema": {
+              "type": "object",
+              "properties": {
+                "total": {
+                  "type": "integer",
+                  "format": "int32",
+                  "description": "number of tags found for user for the given parameters"
+                },
+                "contacts": {
+                  "type": "array",
+                  "items": {
+                    "type": "object",
+                    "properties": {
+                      "type": {
+                        "type": "string"
+                      },
+                      "label": {
```

Comment:
is `label` the translation of `name` ? 

btw, don't you need to declare an id ?